plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -139,15 +139,6 @@ tests =
   , goldenWithEval "Don't panic on {-# UNPACK #-} pragma" "TUNPACK" "hs"
   , goldenWithEval "Can handle eval inside nested comment properly" "TNested" "hs"
   , goldenWithEval "Test on last line insert results correctly" "TLastLine" "hs"
-  , testGroup "with preprocessors"
```

Comment:
hang on, what happened here? Aren't these _broken_ in 8.6? So we should instead just be able to enable them?

Reply:
Oooohh, that would obviously be better, yeah -> d9a2c395e69ef59db70e42b1eb55f4d9fc2730a3